README.md

```diff
@@ -1,6 +1,55 @@
 # curriculum-cr-training-mobile-menu
 
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+> This repo host an initial code for the tasks performed during the Code Reviewers training.
 
 Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+This is code review practice project.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live version
+[live version](https://alexander16108.github.io/CR-mobile-menu/https://alexander16108.github.io/CR-mobile-menu/)
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+Clone the repo with `gh repo clone alexander16108/CR-mobile-menu`

```

Comment:
I think there is an issue with your cloning repo URL. The link is unavailable. Please, check 📝
Providing a wrong link can make a bad impression on you as a developer.